spec/person_spec.rb

```diff
@@ -0,0 +1,25 @@
+require_relative '../person'
+
+describe Person do
+  before do
+    @person = Person.new(20)
+  end
+
+  describe '#initialize' do
+    it 'initializes with an age' do
+      expect(@person.age).to eq(20)
+    end
+  end
+
+  describe '#of_age?' do
+    it 'should return true if the person is of age' do
+      expect(@person.of_age?).to eq(true)
+    end
+  end
+
+  describe '#can_use_services?' do
+    it 'should return true if the person is of age' do
+      expect(@person.can_use_services?).to eq(true)
+    end
+  end
```

Comment:
- [ ] like the previous CR mentioned, in this project you are expected to test all methods in all classes which in this class should include the `add_rental` method.
which might look similar to the below test.
```
context "When adding a new rentals" do

    it "The add_rental method returns a rental and adds its to the book" do
      book = Book.new("Title", "Author")
      person = Person.new(24, "Bob")

      rental = person.add_rental(book, "2021-05-04")

      expect(person.rentals).to eql [rental]
    end
  end

```